content/deploy/restore/keycloak.md

```diff
@@ -0,0 +1,103 @@
+---
+title: 'Restore Keycloak'
+date: '2024-01-08T13:26:23-05:00'
+categories: ["how-to"]
+description: How to restore a Keycloak backup on Rhize
+weight: 300
+menu:
+  main:
+    parent: restore
+    identifier:
+---
+
+This guide shows you how to restore Keycloak in your Rhize environment.
+
+| Restoring Keycloak to an existing running instance will involve downtime
+
+Expected downtime for an existing running instance depends on the cluster performance sizing, backup size and bandwidth speed to the kubernetes cluster. Downtime is typically under a minute.
```

Comment:
```suggestion
{{% notice "caution" %}}

Restoring Keycloak to an existing running instance will involve downtime.

The expected downtime for an existing running instance depends on the cluster performance sizing, backup size and bandwidth speed to the Kubernetes cluster. Downtime is typically under a minute.

{{% /notice %}}
```

For special banners and call-outs, we have a Hugo shortcode to automatically format them depending on the notice level. Check out
[Contributing.md](https://github.com/libremfg/libremfg.github.io/blob/main/CONTRIBUTING.md#notices) for examples.